assets/chef-recipes/cookbooks/debugging_tools/recipes/default.rb

```diff
@@ -1,4 +1,4 @@
-packages = %w[gdb valgrind]
```

Comment:
The binutils is always needed for MaxScale testing whereas GDB and Valgrind are not. Although GDB might be useful if `debug=gdb-stacktrace` is enabled, Valgrind is never used by MaxScale and is thus a useless dependency.

If possible, either move the binutils back into the MaxScale product or provide a separate product for MaxScale that includes only it. The binutils package was previously installed but for some reason it no longer gets pulled in by the CI builds.

Reply:
Created a separate product containing only `binutils` package

Reply:
That'll be perfect, we can add that as a part of the builds we use for testing.